src/main.rs

```diff
@@ -673,12 +670,14 @@ See https://crash-stats.mozilla.org/api/tokens/ for details.\n\n\n",
     let mut socc_json = None;
     let tmp_socc_json2: serde_json::Value;
 
-    if let Some(api_token) = &api_token {
-        // Download the json socorro has for the minidump
-        let socc_output_req = reqwest::Client::new()
-            .get("https://crash-stats.mozilla.org/api/ProcessedCrash/")
-            .header("Auth-Token", *api_token)
-            .query(&[("crash_id", crash_id)]);
+    if let Some(crash_id) = &crash_id {
+        // Download the json socorro has for the minidump. This can be done without an API token.
```

Comment:
this is not actually completely correct. if you don't have appropriate permissions a few fields will be silently omitted and change the results. I think windows module signing info is the major one.